### PR TITLE
New titles, paragraph, text highlights etc

### PR DESCRIPTION
**Pull request by lulen11**
Hi @kbardi 

I've updated a lot of stuff around typography and theming variables in this branch. I think we should merge it before I do too much more but would love your help with compatibility across blueprint versions. (ie. Things such as: I've removed the block, main, and gradient titles because we're no longer using them and I think it's time we go through our projects and refactor the new titles but let me know what you think)

I would **also love your help with...** figuring out how we should implement the **paragraph text highlight** styles. You'll see I've got a lot of comments around these styles because I thought we could add in within the paragraph component but maybe it needs to be it's own component? **[Watch this for more context!](https://www.loom.com/share/59490bc7621e4cd2a29832bb6e7ca203)**

### CODE REVIEW

**Comment by lulen11** [comment]
_Side note around theming:_ - I started this branch before we had merged the new `website` theme in with staging. In this branch, I've treated the `light` theme as that new website theme (setting up featureFonts and colors to suit the new web design) and then once this branch is merged in with master, I will branch off and sort out the themes by transferring all the `light` theme styles over to the `website` theme and then deleting the `light` theme.

File: src/lib/styles/global/fonts.module.scss [hunk 1]
```diff
@@ -24,63 +24,101 @@ $fonts-path: 'https://d2ylaz7bdw65jx.cloudfront.net/assets/fonts';
...
+@font-face {
+  font-family: 'GT Pressura Mono';
+  src: url('#{$fonts-path}/gt-pressura/gt-pressura-mono-regular-webfont.woff2') format('woff2'),
+    url('#{$fonts-path}/gt-pressura/gt-pressura-mono-regular-webfont.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
+
 @font-face {
-  font-family: 'Blackhawk';
```
Comment (kbardi) [inline src/lib/styles/global/fonts.module.scss]: Can we remove Blackhawk? I think we are still using it somewhere...

Comment (lulen11) [reply src/lib/styles/global/fonts.module.scss]: We aren't using it anywhere. In all instances, it will be replaced by Poppins or Luna Emu

**Comment by lulen11** [comment]
@kbardi you've approved this one but there's still the WIP of the text highlight styles. I'll remove them and add them as comments so that we can address them in a new  branch maybe

**Comment by lulen11** [comment]
@kbardi I ran `yarn test:update-snapshot` and finally got it passing locally by commenting out a line (line 38) in `src/lib/components/countdownTimer/countdownTimerUtils.ignore.test.js` ... I didn't understand why this line would cause the tests to fail (because I haven't touched anything to do with that component) but when I commented it out, the tests passed locally. Then once that pushed up, the checks remotely failed. Except for when I go into 'Details' and re-run them, a few then work... I'm super lost with all these tests... I'm going to merge and then hopefully we can address in a new branch off of master. I would LOVE a tests lesson please @kbardi

**Pull request merged**